2021q4/avx-bug.adoc

```diff
@@ -0,0 +1,30 @@
+== AVX bug on amd64
+
+Contact: Konstantin Belousov <kib@FreeBSD.org>
+
+Commit: 73b357be92385cbb70ba19e7023a736af2c6b493
```

Comment:
I have no idea how to do this in new doc format.  If you provide the patch, I will integrate it.

Reply:
I just happened to notice this one; I'll do more thorough review later.
This will reference a git commit in the source repository:
```gitref:d15e810db9a5[repository=src]```